http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -493,6 +504,18 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       else appQuery.updateStatus(dbApp.app.id, AppStatus.Error).transaction
     } yield ()
 
+  def createGalaxyPostresDisk(project: GoogleProject, zone: ZoneName, appName: AppName): Disk =
+    Disk
+      .newBuilder()
+      .setName(s"${appName.value}-${config.galaxyAppConfig.postgresDiskNameSuffix}")
+      .setZone(zone.value)
+      .setSizeGb(config.galaxyAppConfig.postgresDiskSize.gb.toString)
+      .setPhysicalBlockSizeBytes(config.galaxyAppConfig.postgresDiskBlockSize)
+      .build()
+
+  def getGalaxyPostresDiskName(appName: AppName): DiskName =
+    DiskName(s"${appName.value}-${config.galaxyAppConfig.postgresDiskNameSuffix}")
```

Comment:
This will name disks like `saturn-k8s-<uuid>-gxy-postres-disk`

I think this is reasonable, as long as it doesn't exceed name length requirements. :) In the future the UI will name this disk itself.